packages/ui/src/stories/Card.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+import { Card as Component, Typography } from '@open-condo/ui/src'
+
+export default {
+    title: 'Components/Card',
+    component: Component,
+    args: {
+        width: 400,
+        bodyPadding: 24,
+        hoverable: true,
+        children: 'A decision tree is a decision support tool that uses a tree-like model of decisions and their possible consequences, including chance event outcomes, resource costs, and utility. It is one way to display an algorithm that only contains conditional control statements.  Decision trees are commonly used in operations research, specifically in decision analysis, to help identify a strategy most likely to reach a goal, but are also a popular tool in machine learning.',
+    },
+} as ComponentMeta<typeof Component>
+
+const Template: ComponentStory<typeof Component> = ({ children, ...rest }) => {
+    return (
+        <Component
+            {...rest}
+        >
+            <Typography.Paragraph ellipsis={{ rows: 3 }}>
```

Comment:
better to move non-changing object props to constants

Reply:
That's just a story...not component itself, so it's ok to left it as is